assets/scss/hero.scss

```diff
@@ -29,61 +53,85 @@
   }
 
   &.browse-jumbotron {
-    .container {
-      padding-bottom: 3rem;
+    margin-top: 2rem;
+    border-radius: 0.5rem;
+    overflow: hidden;
+    &:before {
+      display: none;
     }
-    &:after {
-      border-bottom: 145px solid $bodygrey;
-      border-left: 60px solid transparent;
-      bottom: 0;
-      content: '';
-      display: block;
-      height: 0;
-      position: absolute;
-      right: 0;
-      width: 0;
-      z-index: 3;
+    .container {
+      .card {
+        margin: 1rem;
+        max-width: 37rem;
+        border: none;
+        color: $black;
+        .btn {
+          text-decoration: none;
+        }
```

Comment:
The button has too much whitespace (top/bottom) compared to the design.

Reply:
fixed